tests/integration/integration_python_test.go

```diff
@@ -2121,3 +2121,17 @@ func installPythonProviderDependencies(t *testing.T, dir string) {
 	out, err := cmd.CombinedOutput()
 	require.NoError(t, err, "output: %s", out)
 }
+
+// Regression test for https://github.com/pulumi/pulumi/issues/18768
+//
+//nolint:paralleltest // ProgramTest calls t.Parallel()
+func TestOrganization(t *testing.T) {
+	integration.ProgramTest(t, &integration.ProgramTestOptions{
+		Dir:          filepath.Join("python", "organization"),
```

Comment:
Should we add such a test for all languages, just so we don't accidentally break one of the others?

Reply:
We should add a better type of test across all languages that would test current CLI + previous SDK, and previous CLI + current SDK. I’ll raise it in the postmortem.

Reply:
> We should add a better type of test across all languages that would test current CLI + previous SDK, and previous CLI + current SDK. I’ll raise it in the postmortem.

Yes please! Pretty sure we've had a few bugs of this type over the years. Just running conformance three times would probably cover most of it.